openml/utils.py

```diff
@@ -31,6 +33,52 @@
     pass
 
 
+def check_task_existence(task_meta_data: dict) -> Union[int, None]:
+    """Checks if any task with exists on test server that matches the meta data.
+
+    Parameter
+    ---------
+    task_meta_data : dict
+        A dictionary containing meta-information on the task fetched from the test server.
+
+    Return
+    ------
+    int, None
+    """
+    return_val = None
+    try:
+        tasks = openml.tasks.list_tasks(output_format="dataframe")
+        tasks = typing.cast(pd.DataFrame, tasks).loc[
+            tasks["task_type"] == task_meta_data["task_type"]
```

Comment:
Could you please add the task type to `list_tasks` to reduce overhead? Also, could you please make the argument the individual elements of `task_meta_data` to make this function easier to read?